Source/Core/Core/PowerPC/PowerPC.h

```diff
@@ -57,7 +57,7 @@ struct tlb_entry
 };
 
 // This contains the entire state of the emulated PowerPC "Gekko" CPU.
-struct GC_ALIGNED64(PowerPCState)
+struct alignas(16) PowerPCState
```

Comment:
Probably someone wanted a stricter alignment to force this structure to start on a 64 byte cacheline. 

The only actual alignment requirement is the paired single registers must be 16 byte aligned.